File: Data Structures & Algorithms/is-anagram/submission-18.cpp
#include <string>

class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length()) return false;
        int letters[26];

        for (int i {}; i  <  (int)s.length(); ++i) {
            letters[(int)(s[i] - 97)]++;
            letters[(int)(t[i] - 97)]--;
       }

       for (int num: letters) {
        if (num != 0) return false;
       }

       return true;
    }


};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-18.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if not nums:
            return 0

        max_count = 1

        start_set = set()

        for num in nums: start_set.add(num)

        for num in nums:
            if (num -1 not in start_set):
                current_count = 1
                while (num + 1 in start_set):
                    current_count += 1
                    num += 1
            
                if current_count > max_count:
                    max_count = current_count
        
        return max_count    
